proposals/i18n-getOSLanguage.md

```diff
@@ -0,0 +1,74 @@
+# Proposal: i18n.getOSLanguage>
+
+**Summary**
+
+Allow to get the language of your operating system as an `IETF tag`
+
+**Document Metadata**
+
+**Author:** carlosjeurissen
+
```

Comment:
This dropped the "sponsoring browser" item [here](https://github.com/w3c/webextensions/blob/e02ace0c34a17ae99a446c6e73fb4db778f41a53/proposals/proposal_template.md?plain=1#L19).

That was added to ensure the proposals here don't become a graveyard of aspirational APIs and instead reflect ones that at least one browser is committed to implementing in the immediate future.

I'll need to do some quick investigation into the implementation.  If it looks straightforward, I'm comfortable putting Google Chrome as the sponsoring browser.

Reply:
Added back the Sponsoring browser with a TBD.

Reply:
I'm supportive of sponsoring for Safari and plan to implement it as soon as we have consensus.